source/jsonParsing.ml

```diff
@@ -76,13 +76,13 @@ module JsonAst = struct
       line: int;
       column: int;
     }
-    [@@deriving equal, show]
+    [@@deriving equal, show, compare, sexp_of]
```

Comment:
Do we really need `sexp_of`? Is it because we need it to be `Hashable`? In that case you can use `deriving hash`?